### ISSUE TITLE
NTR: Methylsterols high in cultured lymphoblasts or fibroblasts (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8278]
New term request Methylsterols high in cultured lymphoblasts or fibroblasts (elthat_110328054836):

**Comment by pnrobinson** [comment]
@MickeySegal -- similar question as in https://github.com/obophenotype/human-phenotype-ontology/issues/8278

**Comment by pnrobinson** [comment]
@MickeySegal
what is the clinical context and what are the specific molecules?

**Comment by MickeySegal** [comment]
https://www.omim.org/entry/616834
Microcephaly, congenital cataract, and psoriasiform dermatitis (MCCPD) represents an inborn error of cholesterol metabolism that is characterized by accumulation of a large amount of methylsterols, particularly dimethylsterols, in affected patients. ...  cholesterol biosynthesis defect was suspected, and plasma sterol analysis showed a 20-fold increase in 4-alpha-monomethyl sterols and a 500-fold increase in 4,4-prime-dimethyl sterols. ...Methylsterols were markedly elevated in patient fibroblasts compared to controls

https://www.ncbi.nlm.nih.gov/books/NBK51754/
When cultured in cholesterol-depleted medium, lymphoblastoid cells of individuals with CHILD syndrome and CK syndrome have increased levels of methyl- and carboxy-sterols and slightly decreased levels of cholesterol

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001113
- label: Elevated circulating methysterol concentration
- parents: [Abnormal circulating lipid concentration (HP:0003119)](https://hpo.jax.org/app/browse/term/HP:0003119)
- pmid: 
- def: The concentration of a methysterol in the blood circulation is above the upper limit of normal.
- comment: This group of compounds includes thje 4-alpha-monomethyl sterols and 4,4-prime-dimethyl sterols.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8278
- synonyms:
